specification/automation/resource-manager/readme.md

```diff
@@ -156,41 +158,77 @@ input-file:
-- Microsoft.Automation/stable/2015-10-31/jobSchedule.json
-- Microsoft.Automation/stable/2015-10-31/linkedWorkspace.json
-- Microsoft.Automation/stable/2015-10-31/module.json
-- Microsoft.Automation/stable/2015-10-31/schedule.json
-- Microsoft.Automation/stable/2015-10-31/variable.json
-- Microsoft.Automation/stable/2015-10-31/webhook.json
-- Microsoft.Automation/stable/2015-10-31/watcher.json
-- Microsoft.Automation/preview/2017-05-15-preview/softwareUpdateConfiguration.json
-- Microsoft.Automation/preview/2017-05-15-preview/softwareUpdateConfigurationRun.json
-- Microsoft.Automation/preview/2017-05-15-preview/softwareUpdateConfigurationMachineRun.json
-- Microsoft.Automation/preview/2017-05-15-preview/sourceControl.json
-- Microsoft.Automation/preview/2017-05-15-preview/sourceControlSyncJob.json
-- Microsoft.Automation/preview/2017-05-15-preview/sourceControlSyncJobStreams.json
-- Microsoft.Automation/preview/2017-05-15-preview/job.json
-- Microsoft.Automation/stable/2018-01-15/dscNode.json
-- Microsoft.Automation/stable/2018-01-15/dscCompilationJob.json
-- Microsoft.Automation/stable/2018-01-15/dscNodeConfiguration.json
-- Microsoft.Automation/stable/2018-01-15/dscNodeCounts.json
-- Microsoft.Automation/stable/2018-06-30/runbook.json
-- Microsoft.Automation/stable/2018-06-30/python2package.json
 - Microsoft.Automation/preview/2020-01-13-preview/privateEndpointConnection.json
 - Microsoft.Automation/preview/2020-01-13-preview/privateLinkResources.json
+- Microsoft.Automation/preview/2020-01-13-preview/python2package.json
+- Microsoft.Automation/preview/2020-01-13-preview/dscNode.json
+- Microsoft.Automation/preview/2020-01-13-preview/dscNodeConfiguration.json
+- Microsoft.Automation/preview/2020-01-13-preview/dscCompilationJob.json
+- Microsoft.Automation/preview/2020-01-13-preview/dscNodeCounts.json
+- Microsoft.Automation/preview/2020-01-13-preview/sourceControl.json
+- Microsoft.Automation/preview/2020-01-13-preview/sourceControlSyncJob.json
+- Microsoft.Automation/preview/2020-01-13-preview/sourceControlSyncJobStreams.json
+- Microsoft.Automation/preview/2020-01-13-preview/account.json
+- Microsoft.Automation/preview/2020-01-13-preview/certificate.json
+- Microsoft.Automation/preview/2020-01-13-preview/connection.json
+- Microsoft.Automation/preview/2020-01-13-preview/connectionType.json
+- Microsoft.Automation/preview/2020-01-13-preview/credential.json
+- Microsoft.Automation/preview/2020-01-13-preview/hybridRunbookWorkerGroup.json
+- Microsoft.Automation/preview/2020-01-13-preview/jobSchedule.json
+- Microsoft.Automation/preview/2020-01-13-preview/linkedWorkspace.json
+- Microsoft.Automation/preview/2020-01-13-preview/module.json
+- Microsoft.Automation/preview/2020-01-13-preview/schedule.json
+- Microsoft.Automation/preview/2020-01-13-preview/variable.json
+- Microsoft.Automation/preview/2020-01-13-preview/watcher.json
+- Microsoft.Automation/stable/2019-06-01/dscConfiguration.json
```

Comment:
Same above, do you still need to mix api versions in one sdk?

Reply:
Similar reasoning as for  2019-06-01 api-version.